packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/ImageEdit.ts

```diff
@@ -414,7 +437,14 @@ export default class ImageEdit implements EditorPlugin {
      */
     private updateWrapper = (context?: DragAndDropContext) => {
         const wrapper = this.wrapper;
-        if (wrapper) {
+        if (
+            wrapper &&
+            this.editInfo &&
+            this.image &&
+            this.clonedImage &&
+            this.options &&
+            this.shadowSpan?.parentElement
```

Comment:
In theory these info should either all exist or none exists.

Can we create an interface for them and create the instance together, so that we only need to check if that instance exist.

For example:
```ts
interface EditingImageInfo {
   image: HtmlImageElement;
   clonedImage: HtmlImageElement;
   ...
}

class ImageEdit {
   private edtingImageInfo: EditingImageInfo | null = null;
   ..


  this.editingImageInfo = {
   ...
  }


  ...


  if (this.editingImageInfo) {
     ...
 }
 
...

this.editingImageInfo = null;
}
```

Reply:
How do we add the elements in this object? Some of them are created in different parts in the code.

Reply:
This needs some kind of refactoring.

If that is too complex, feel free to do it in separate PR.